src/Components/Product/Filter.js

```diff
@@ -0,0 +1,67 @@
+function Filter() {
+  return (
+    <div className="filter-container">
```

Comment:
id this is going to be a sidebar then you could use <aside> instead of <div> for semantics

Reply:
I'm keeping this as just a component and not really a sidenav, that's why I omitted it.